wuvt/admin/views.py

```diff
@@ -492,6 +492,29 @@ def donation_index():
         if 'reset_stats' in request.form:
             redis_conn.set('donation_stats_start',
                            datetime.datetime.utcnow().isoformat())
+        if 'set_paid' in request.form:
+            # TODO log who set as paid
+            o = Order.get_or_404(id=request.form["id"])
+            o.set_paid(method="check")
+            db.session.add(o)
+            try:
+                db.session.commit()
+            except:
+                db.session.rollback()
+                raise
+
+        # TODO support setting asynchronously, rather than redirecting
+        if 'set_shipped' in request.form:
+            # TODO log who set as paid
+            o = Order.get_or_404(id=request.form["id"])
```

Comment:
```
AttributeError: type object 'Order' has no attribute 'get_or_404'
```

This should be `o = Order.query.get(request.form["id"])`

Reply:
ah, right. or `.query.get_or_404`, but that may not make sense here...